script.js

```diff
@@ -0,0 +1,75 @@
+const addBtn = document.querySelector('.add-btn');
+const booksContainer = document.querySelector('.books-container');
+let books = [];
+
+// retrive books from local storage (if they exist) when page loads
```

Comment:
- [ ] Great job.. everything works as expected. kindly remove comments for clean code practices

Reply:
I don't understand, since when did it become a bad practice to write comments on your code?